src/backend/src/services/statistics.services.ts

```diff
@@ -115,4 +117,15 @@ export default class StatisticsService {
       };
     });
   }
+
```

Comment:
can you add javadoc here. basically if you do /** before a function it should autogenerate with the parameters you need to fill in.